index.html

```diff
@@ -92,19 +161,44 @@ <h3 id="the-difficulties-of-ruby-processing">The Difficulties of Ruby Processing
     whether it can be allowed to be laid over the characters preceding or following,
     and whether this affects the position of the base characters</p>
     <aside class=note title="Protrusion of ruby from base characters">
-      In movable type typesetting,
-      ruby characters protruding from their base characters
-      may not hang over neighboring kanji,
-      but often were allowed to hang over neighboring kana.
-      However, when the ruby is katakana,
-      some publishers would set it so that
-      it would not hang over the kana neighboring the base character.
-      Also, when the characters around the base characters
-      were kanji on one side and kana on the other side,
-      for the sake of balance
-      the ruby would sometimes be set so as to hang
-      neither over the kanji nor over the kana
-      (which would therefore both be spaced away from the base character).
+      <span class="checkme">
+      From principle not to have extra spacing in Japanese composition, 
```

Comment:
Maybe "extra spacing between base characters" is more clear than "extra spacing"?

Reply:
ah,, let me change as "extra spacing between characters".